.github/workflows/check-full.yml

```diff
@@ -0,0 +1,57 @@
+name: Code Testing
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '42 1 * * *'
+
+jobs:
+  codecheck:
+    runs-on: ubuntu-latest
+    name: Run codecheck
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Install virtualenv + python3-setuptools
+        run: sudo apt-get install virtualenv python3-setuptools
+
+      - name: Set up Python virtualenv environment
+        run: virtualenv -p /usr/bin/python3 venv3
```

Comment:
Not sure how long the virtualenv stuff takes, but given this runs in a throwaway VM, it should be possible to skip venv.

Reply:
Ah right. But with the virtualenv approach we'd always get the latest version via pip instead of the version that's available from ubuntu-latest, nor?